ui/src/utils/object.js

```diff
@@ -1 +1,19 @@
-export const isEmpty = obj => !obj || !Object.keys(obj).length;
+export const isEmpty = (obj) => !obj || !Object.keys(obj).length;
+
+// Creates a copy of original object and removes specific keys from it
+export const stripKeys = (obj, keys, deep = true) =>
```

Comment:
Not gonna lie, took me some time to understand this function. Not sure about the conventions here but I find that deeply nested ternary operators are extremely hard to read.

Reply:
IMO splitting into if else branches would have been easier to process, something like this:

```javascript
const stipKeysV2 = (obj, keys, deep = true) => {
    if (obj !== Object(obj)) return obj

    if (Array.isArray(obj)) {
        if (!deep) return obj
        return obj.map((item) => stripKeys(item, keys, deep))

    return Object.keys(obj)
      .filter((k) => !keys.includes(k))
      .reduce(
        (acc, x) =>
          Object.assign(acc, {
            [x]: deep ? stripKeys(obj[x], keys, deep) : obj[x],
          }),
        {}
      );
}
```

Reply:
Done